deploy/charts/rook-ceph/templates/resources.yaml

```diff
@@ -8873,6 +8906,15 @@ spec:
         - jsonPath: .status.phase
           name: Phase
           type: string
+        - jsonPath: .status.info.endpoint
+          name: Endpoint
+          type: string
+        - jsonPath: .status.info.secureEndpoint
+          name: Secure Endpoint
+          type: string
+        - jsonPath: .metadata.creationTimestamp
+          name: Age
+          type: date
```

Comment:
Does this `date` type show the full timestamp, or the age?

Reply:
This is indeed the age, not a full timestamp as shown in the example provided above